frontend/app/[team]/apps/[app]/environments/[environment]/[[...path]]/page.tsx

```diff
@@ -503,7 +503,7 @@ export default function Environment({
 
     const a = document.createElement('a')
     a.href = url
-    a.download = `${environment.name}.env`
+    a.download = `${environment.app.name} [${environment.name}].env`
```

Comment:
let's avoid adding a space to filenames. it can make working with files in a program or a shell environment tricky. use `-` instead

```
λ cat Downloads/app \ development.png 
```

Reply:
@rohan-chaturvedi fixed.

`feat: lowercase app and env name, use `.` as a separator instead of a space`